### PR TITLE
RUL-79 Add colors and styles for tags

### PR DESCRIPTION
**Pull request by EvrardCaron**
**Description (for Contributor and Core Developer)**
Added colors and styles for tags

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/base/variables.less [hunk 1]
```diff
@@ -50,6 +50,34 @@
 @AknDefaultFontColor: @AknSlateGrey;
 @AknFormFontColor: @AknDarkBlue;
 
+@AknIllustrationPurple20: #f3eef9;
```
Comment (mmetayer) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/base/variables.less:53]: why "Illustration"?

Comment (EvrardCaron) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/base/variables.less:53]: This has been decided by Stephane to make sure not to confuse them with the current colors defined.

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/components/Tag.less [hunk 2]
```diff
@@ -0,0 +1,54 @@
+@import '../base/variables';
+
+.AknTag {
+  height: 18px;
+  line-heigh: 16px;
```
Comment (pierallard) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/components/Tag.less:5]: This should not work

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/components/Tag.less [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  font-size: @AknFontSizeSmall;
+  text-transform: uppercase;
+  border: 1px solid;
+  padding: 0 6px;
+  border-radius: 2px;
+
+  &--default,
```
Comment (pierallard) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/less/components/Tag.less:12]: By convention, in BEM, you should not have a "--default". Do not add --default IS the default. These properties should be directly at line 11, to be displayed by default.

**Pull request merged**